eng/Build.props

```diff
@@ -205,6 +205,8 @@
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.CodeAnalysis.CSharp.Workspaces.4.5.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.CodeAnalysis.VisualBasic.Workspaces.4.5.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\textOnlyPackages\src\**\Microsoft.CodeAnalysis.4.5.0.csproj" />
+
+    <DependencyPackageProjects Include="$(RepoRoot)src\textOnlyPackages\src\**\System.ComponentModel.Composition.7.0.0.csproj" />
```

Comment:
@Evangelink - Apologies, I noticed it was missing from the Workflow instructions I pointed you too so I added it for you. 
 I opened a [PR](https://github.com/dotnet/source-build-reference-packages/pull/671) to address the workflow.